lib/winapi/badptr.c

```diff
@@ -0,0 +1,9 @@
+#include <windows.h>
+
+BOOL IsBadWritePtr(LPVOID lp, UINT_PTR ucb)
```

Comment:
nit: Other winapi functions have a space between function name and parameter parentheses.

Reply:
Should have been fixed.